packages/steps-s3-copy/src/steps-s3-copy-construct.ts

```diff
@@ -412,6 +412,10 @@ export class StepsS3CopyConstruct extends Construct {
         new ServicePrincipal("ecs-tasks.amazonaws.com"),
         new ServicePrincipal("lambda.amazonaws.com"),
       ),
+      // Allow the task definition role ecr access to the guardduty agent
+      // https://docs.aws.amazon.com/guardduty/latest/ug/prereq-runtime-monitoring-ecs-support.html#before-enable-runtime-monitoring-ecs
+      // Which is in another account - 005257825471.dkr.ecr.ap-southeast-2.amazonaws.com/aws-guardduty-agent-fargate
+      managedPolicies: [ManagedPolicy.fromAwsManagedPolicyName('service-role/AmazonECSTaskExecutionRolePolicy')]
     });
 
     // the role is assigned to lambdas - so they need enough permissions to execute
```

Comment:
Actually the lambda Managed Role is also added here.. can you unify it so they are both done the same way..

Either both as addMangedPolicy() - or both in the array.